app/src/main/java/ch/epfl/sdp/game/Game.java

```diff
@@ -108,11 +114,42 @@ public void destroyGame() {
     public void update() {
```

Comment:
Can you also write a javadoc?

Reply:
Since it's an overriden method, I don't think it's useful

Reply:
And in fact, there already is one

Reply:
And in fact, there already is one